kapitan/cli.py

```diff
@@ -430,6 +439,32 @@ def ref_write(args, ref_controller):
         secret_obj = GoogleKMSSecret(data, key, encode_base64=args.base64)
         tag = '?{{gkms:{}}}'.format(token_path)
         ref_controller[tag] = secret_obj
+#
+# Azure KMS marker -- remove it
```

Comment:
`remove it` ?

Reply:
sorry, forgot abt that, my visual clues, will remove it